tests/checks/cel.yml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: app1
```

Comment:
the name should rather be "fire" or "forbidden-annotation"

Reply:
fixed